crates/core/app.rs

```diff
@@ -1536,7 +1536,7 @@ put into a URI. Note that a path is guaranteed to start with a */*.
 
 *{host}*: Optional. This is replaced with your system's hostname. On Unix,
 this corresponds to calling *gethostname*. On Windows, this corresponds to
-calling *GetComputerNameExW* to fetch the system's \"physical DNS hostname.\"
+calling *GetComputerNameExW* to fetch the system's \"physical DNS hostname\".
```

Comment:
I had it this was when I first wrote it, but it looked wrong to me (I'm in the US). Can you change it back please? And in the place above too.

Reply:
Oh, I didn't know this is the way it's written in the US. I reverted it.

Reply:
Yeah it's annoying. I sometimes put the period outside the quotes anyway.